### PR TITLE
Make sure variables are either static or declared externally

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2/core/textinputbuffer.c [hunk 1]
```diff
@@ -108,7 +108,7 @@ void textinputbuffer_cursor_right(textinputbuffer * tib)
             selectionOffset++;
         } while (!utf8_is_codepoint_start(ch) && selectionOffset < selectionMaxOffset);
 
-        tib->selection_size = max(0, tib->selection_size - (selectionOffset - tib->selection_offset));
+        tib->selection_size = tib->selection_size - (selectionOffset - tib->selection_offset);
```
Comment (AaronVanGeffen) [inline src/openrct2/core/textinputbuffer.c]: Unrelated?

File: src/openrct2/world/mapgen.c [hunk 2]
```diff
@@ -61,12 +61,7 @@ const char *GrassTrees[] = {
     "TEL     ",     // European Larch Tree
 };
 
-// Trees to be placed in proximity to water
-const char *GrassWaterTrees[] = {
-    "TWW     "      // Weeping Willow Tree
```
Comment (AaronVanGeffen) [inline src/openrct2/world/mapgen.c]: Why has this block been removed?

Comment (janisozaur) [reply src/openrct2/world/mapgen.c]: It was unused.

File: src/openrct2/core/textinputbuffer.c (hunk 1 above)
Comment (janisozaur) [inline src/openrct2/core/textinputbuffer.c]: Right, that's from other PR: https://github.com/OpenRCT2/OpenRCT2/pull/6421

**Comment by janisozaur** [comment]
Depends on https://github.com/google/googletest/pull/1045
